### ISSUE TITLE
RUNTIME_SIGALRM_HANDLER_TRIAGED causes missed 

### ISSUE THREAD

**Issue by bushidocodes** [gwsystems/sledge-serverless-framework#227]
RUNTIME_SIGALRM_HANDLER_TRIAGED is a variant of the EDF scheduler that attempts to triage which workers a might actually preempt based on deadline in order to reduce spurious SIGARLMs. However, now that the SIGALRM handler is also responsible for checking the thread local epoll for data, this optimization might result in sandboxes blocking unexpectedly long.

A possible solution is to add a runtime array of the number of sandboxes in the blocked state per worker. If a worker has a non-zero value, it should always be forwarded the SIGALRM such that it checks its epoll handler each quantum.

**Comment by emil916** [comment]
The proposed solution sounds promising, but what if a worker has a NZ value (hence there's a blocked sandbox at that worker) AND meanwhile the worker picked another sandbox from its local queue (say the global head was further in time) and began executing that sandbox, then should the worker really be forwarded the SIGALRM? Just thinking out loud...

**Comment by bushidocodes** [comment]
Closing as obsolete — this was resolved by the architecture change in #365, and the premise no longer holds.

### Why it no longer applies

This issue was about the **per-worker epoll** being polled from the SIGALRM handler: at the time, each worker had its own thread-local epoll checked via `scheduler_execute_epoll_loop()` inside the SIGALRM/scheduler path. Under `RUNTIME_SIGALRM_HANDLER_TRIAGED`, a worker whose current sandbox wouldn't be preempted (`scheduler_worker_would_preempt()` false) never received the SIGALRM, so it never ran its epoll loop, and a sandbox blocked on I/O on that worker could stall. That coupling was the bug.

**#365 ("remove epoll from workers", commit `beed67c`) removed the entire per-worker epoll mechanism**, deleting:
- `runtime/include/scheduler_execute_epoll_loop.h` (the epoll poll in the SIGALRM/scheduler path)
- `runtime/include/worker_thread_epoll.h` / `worker_thread_epoll_file_descriptor`

Epoll is now owned exclusively by the dedicated **listener thread** (the only `epoll_wait` in the tree). When a sandbox would block on I/O, its `on_eagain` callback re-registers the session with the listener's epoll (`listener_thread_register_http_session`), and the listener re-enqueues it to the global request scheduler once it's ready.

So the worker SIGALRM (and the TRIAGED triage) is now purely about **preemption** — switching to an earlier-deadline sandbox — and is fully decoupled from I/O readiness. TRIAGED can no longer cause missed epoll updates, and the proposed per-worker "blocked sandbox count" forwarding (and @emil916's follow-up nuance about a worker running a different sandbox) is no longer needed.

### Verification

Confirmed `beed67c` (#365) is in `master`, and `grep` finds zero remaining references to the worker epoll. Ran the current runtime in the exact affected mode (`SLEDGE_SIGALRM_HANDLER=TRIAGED SLEDGE_SCHEDULER=EDF`, 4 workers):
- fib(10/20/25/30) all correct
- 60 concurrent mixed-deadline requests completed in ~0.7s with 0 timeouts and 0 failures — no "unexpected blocking"

No code change required; closing as resolved by #365.
